examples/gds/client.c

```diff
@@ -0,0 +1,126 @@
+/* This work is licensed under a Creative Commons CCZero 1.0 Universal License.
+ * See http://creativecommons.org/publicdomain/zero/1.0/ for more information. */
+/*
+ * A simple server instance which registers with the discovery server (see server_lds.c).
+ * Before shutdown it has to unregister itself.
+ */
+
+#include "open62541.h"
```

Comment:
You need to rebase onto master and then use the separate files. We switched from amalgamated header to separate headers in the examples. See e.g.: https://github.com/open62541/open62541/blob/master/examples/client.c#L4